examples/conf_example.c

```diff
@@ -37,9 +46,13 @@ int main()
      * OPENSSL_CONF). This will load wolfEngine and make it the default engine
      * for all the algorithms it provides. See wolfEngine.conf for more.
      */
+#if OPENSSL_VERSION_NUMBER < 0x10100000
+    OPENSSL_config(NULL);
```

Comment:
I do get a warning during compilation on this:

```
conf_example.c: In function ‘main’:
conf_example.c:50:5: warning: implicit declaration of function ‘OPENSSL_config’; did you mean ‘OPENSSL_init’? [-Wimplicit-function-declaration]
   50 |     OPENSSL_config(NULL);
      |     ^~~~~~~~~~~~~~
      |     OPENSSL_init
```

Looks like we need an `#include <openssl/conf.h>`?